global.json

```diff
@@ -1,10 +1,12 @@
 {
   "tools": {
-    "dotnet": "9.0.100"
+    "dotnet": "9.0.100",
+    "xcopy-msbuild": "17.10.0-pre.4.0"
```

Comment:
Isn't this a little old? And why is it a pre-version?

Reply:
When I added this line on August 29th, it was the latest version, published in April. Looks like there is a newer version that was published in November. Will update: https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-eng/NuGet/Microsoft.DotNet.Arcade.MSBuild.Xcopy/versions/17.12.0

Reply:
This has been updated.